docs/docs/target/kubernetes.md

```diff
@@ -30,6 +30,12 @@ Kubernetes resource definition is scanned for:
 
 Trivy follows the behavior of the `kubectl` tool as much as possible.
```

Comment:
not relevant anymore

Reply:
@itaysk this PR is on merge queue, I'll create a new PR for this change

Reply:
this is the new PR #6608